### ISSUE TITLE
Add focus styles to header elements

### ISSUE THREAD

**Issue by TrulyFlawed** [TrulyFlawed/trulyflawed.github.io#7]
Should be a straightforward accessibility improvement; leaving here as a reminder for later.
